README.md

```diff
@@ -591,10 +591,10 @@ you proceed to [step 4](#verify-itts-client-version).
 
 #### 9.1. Event log and measurements
 
-One of the key components remote attestation is based on is the runtime measurement values (stored in RTMRs
-registers for each TD by the TDX module). These RTMR values are computed from the digests of the entries of
-the TD boot event log. During the system's boot, each event's digest (SHA384) is extended (appended) to the
-current value of the associated RTMR. The resulting value's digest becomes the new value of the RTMR.
+One of the key components remote attestation is the runtime measurement values (stored in RTMRs
```

Comment:
Add "of" after "componenets"

Reply:
"One of" implies a singular subject, but "values" is plural. Might be better to just flip the sentence around and say something like, "Runtime measurement values (paren contents) are key components of remote attestation."

Reply:
Done 57bef18762575d7cfbabc876ec221d46df674c1e